app/src/main/java/com/oracle/javafx/scenebuilder/app/menubar/MenuBarController.java

```diff
@@ -781,19 +783,19 @@ public String getTitle() {
 
         toggleLibraryPanelMenuItem.setUserData(
                 new DocumentControlActionController(DocumentControlAction.TOGGLE_LIBRARY_PANEL) {
-                    @Override
-                    public String getTitle() {
-                        final String titleKey;
-                        if (documentWindowController == null) {
-                            titleKey = "menu.title.hide.library.panel";
-                        } else if (documentWindowController.isLibraryPanelVisible()) {
-                            titleKey = "menu.title.hide.library.panel";
-                        } else {
-                            titleKey = "menu.title.show.library.panel";
-                        }
-                        return I18N.getString(titleKey);
-                    }
-                });
+            @Override
```

Comment:
I think this is auto-formatting working again...

Reply:
> Well, it may require that you rebuild the kit module. Just run `mvn clean install` for the whole project and then run `mvn javafx:run -f app`. I had a similar case and rebuilding kit worked fine in my case.

Yup, that worked. I'll remember that process for the future! :-)